AlphaWallet/Assets.xcassets/add_hide_tokens.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
1. You mean make it like this?
<img width="404" alt="Screenshot 2020-03-30 at 17 13 30" src="https://user-images.githubusercontent.com/55975226/77933639-afaaa800-72b7-11ea-8683-a7d7d5854cbd.png">


5. I don’t know, need debud this case.
7. When I tried to add this token I even don’t see it, it doesn’t appears in any tab, and I don’t see it in notifySubscribersOfUpdatedTokens
8. You mean add table view top inset or increase table view header height?
10. For now I didn’t find any ways to do that.

Reply:
1. No. Here's a screenshot of how it looks like in the Wallet tab. If you compare it to the screenshot above which I sent previously, focus on the sections I highlighted with the red "circle". There are 2 colors (grayish and white) in between the cells. Can you make the version in Add/Hide Tokens be like the one in the Wallet tab? 

<img src="https://user-images.githubusercontent.com/56189/77977984-7748af80-7333-11ea-9278-35c3a46f898b.png" width=230>

Bullet/Numbering is slightly off here, I'll stick to my original version in case I address the wrong question:

7. Did you tap on the + button in the Add/Hide Token screen (previously in Wallet tab) to bring up the screen with the title "Add Custom Token" to do this? This is the `NewTokenViewController` class.
8. I'm referring to this part. Note that this is 12.4, not 13.x. It looks ok in 13.x. Can you remove that additional margin?

<img width="230" src="https://user-images.githubusercontent.com/56189/77979885-6ea6a800-7338-11ea-8c57-1732710c786b.png">

9. If:

> For now I didn’t find any ways to do that.

you are referring to this:

> When we move a token from Hidden to Displayed, can we add it to the bottom of Displayed Tokens instead of to the top?

Can't this be done by setting the `sortIndex` accordingly?

Reply:
Questions #9 I was wrong, the question was about 10. 
   i am not shure. sortIndex will not helo in this case.  As i understand correctrly, you want delete rows without confirmation, just by pressing "minus button", so i didn't find any ways to implement this, only with confirmation. If you know how to implement this, let me know.

7. yes i know how to add tokens, i tried to add token, and this feature works ok

Reply:
> 9. I was wrong, the question was about 10.
> i am not shure. sortIndex will not helo in this case. As i understand correctrly, you want delete rows without confirmation, just by pressing "minus button", so i didn't find any ways to implement this, only with confirmation. If you know how to implement this, let me know.

Ah. Don't worry about it then. Just leave it as that.

> 
> 1. yes i know how to add tokens, i tried to add token, and this feature works ok

Yes, so I was referring to adding the contract 0x8C0Edb69eBF038BA0c7a4873E40fC09725064c2e via this route here:

> Can you add this token on mainnet? 0x8C0Edb69eBF038BA0c7a4873E40fC09725064c2e. It should always have a balance of 1 I observe 2 issues with it. In the Add/Hide tokens screen, it is displayed as a balance of 0 (Wallet tab shows 1). And the "| Issuer: AlphaWallet" text is appear in Add/Hide tokens screen when it doesn't in the Wallet tab